walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -13,8 +13,7 @@ <h1>Laura's Portfolio</h1>
       <img src="images-me/IMG_0469.JPG"/>
       <h2>Projects</h2>
       <ul>
-	<li> Worked at the University of Catalonia on an Autism related project.
-	<li>  
+	<li> Worked at the University of Catalonia on an Autism related project.</li>  
```

Comment:
Still some extra space after the <li>

Reply:
ohh I think I got it now, you mean that it should be <ul><li>text</li></ul>?

Reply:
ohhh I think I got it now, do you mean between the ul closing tag and the p opening tag? (it is my first HTML tag so I am still getting used to it :)

Reply:
Oh no no sorry! I didn't notice that github formatted my list tag into a bullet point haha. What you had before was fine, I was just meaning that the space before "Worked" should be removed

Reply:
Ohh hahaha, got it! I think it should be okay now, let me know if there is anything else I should change.